src/modules/task-dynamic-ui/task-dynamic-ui.html

```diff
@@ -1,11 +1,11 @@
 <template>
   <require from="./task-dynamic-ui.css"></require>
-  <require from="../dynamic-ui-wrapper/dynamic-ui-wrapper"></require>  
-
+  <require from="../dynamic-ui-wrapper/dynamic-ui-wrapper"></require>
+<div class="dynamic-ui--wrapper">
```

Comment:
Wrong indendation.
Also `wrapper` is no modifier of the block `dynamic-ui`, my suggestion:

```suggestion
  <div class="dynamic-ui-wrapper">
```